src/OrbitGl/FrameTrack.cpp

```diff
@@ -79,14 +79,14 @@ FrameTrack::FrameTrack(CaptureViewElement* parent, TimeGraph* time_graph, TimeGr
                              TriangleToggle::InitialStateUpdate::kReplaceInitialState);
 }
 
-float FrameTrack::GetHeaderHeight() const { return 0.f; }
+float FrameTrack::GetHeaderHeight() const { return layout_->GetTrackTabHeight(); }
 
 float FrameTrack::GetHeight() const {
-  return GetMaximumBoxHeight() + layout_->GetTrackBottomMargin();
+  return GetHeaderHeight() + GetMaximumBoxHeight() + layout_->GetTrackBottomMargin();
 }
 
 float FrameTrack::GetYFromTimer(const TimerInfo& /*timer_info*/) const {
-  return pos_[1] - GetMaximumBoxHeight();
+  return pos_[1] - GetHeaderHeight() - GetMaximumBoxHeight();
```

Comment:
This (and the overriding `GetYFromTimer`) looks like it would be more straightforward if we renamed it to something like `GetYOffsetFromTimer`, have it return a position relative to the start of the "Timer Area", and have the caller do the required math to convert this into an absolute position, right?

It's probably not something to change in this PR though, but we should keep this in mind when moving to relative positioning.

Reply:
That would change completely as soon as we have the ContentPane/TimerPane's. In this case, the code would look like before this change, as the pane is already positioned and all computations only need to be relative to its own position.I would keep it as it for now, as this will change anyways soon.